my/location/common.py

```diff
@@ -0,0 +1,17 @@
+from datetime import date, datetime
+from typing import Union, Tuple, NamedTuple, Optional
+
+from my.core import __NOT_HPI_MODULE__
+
+DateIsh = Union[datetime, date, str]
+
+LatLon = Tuple[float, float]
+
+
+# TODO: add timezone to this? can use timezonefinder in tz provider instead though
+class Location(NamedTuple):
+    lon: float
```

Comment:
Kinda nit, but it's inconsistent with `LatLon` above?
Not sure if should change, was never sure which one is right, ran into this recently https://news.ycombinator.com/item?id=30228981 and still confused haha. So just flagging in case you have an opinion